src/black/report.py

```diff
@@ -10,6 +13,12 @@
 from black.output import out, err
 
 
+def root_relative(path: Path, root: Optional[Path]) -> Path:
+    if not root:
+        return path
+    return Path(os.path.relpath(path, start=Path(f"{root}/../")))
```

Comment:
`pathlib.Path` does have `p.relative_to`. But this works fine too 👍

Reply:
```suggestion
    return path.relative_to(root.parent)
```
would be the form I guess. So why the parent?

Reply:
Oh I guess we want the root folder to be included?

Reply:
Yeah

Reply:
The pathlib version is much cleaner to read as well, so maybe we should use it instead.